public/controllers/management/components/management/ruleset/ruleset-editor.js

```diff
@@ -112,45 +117,67 @@ class WzRulesetEditor extends Component {
       try {
         await validateConfigAfterSent();
       } catch (error) {
-        const toast = {
-          title: 'File content is incorrect.',
-          toastMessage: `The content of the file ${name} is incorrect. There were found several error while validating the configuration.`,
-          toastLifeTimeMs: 5000,
+        const options = {
+          context: errorContext,
+          level: UI_LOGGER_LEVELS.ERROR,
+          severity: UI_ERROR_SEVERITIES.BUSINESS,
+          error: {
+            error: error,
+            message:`The content of the file ${name} is incorrect. There were found several error while validating the configuration: ${error.message || error}`,
```

Comment:
spell **errors** `There were found several errors while validating the configuration`